openmrs/apps/registration/app.json

```diff
@@ -106,9 +106,9 @@
             },
             "patientSearch": {
                 "address": {
-                    "label": "Localité/Location",
-                    "placeholder": "Entrer la localité / Enter the Location",
-                    "field": "address2"
+                    "label": "REGISTRATION_SEARCH_ADDRESS_LABEL_KEY",
+                    "placeholder": "REGISTRATION_SEARCH_ADDRESS_PLACEHOLDER_KEY",
+                    "field": "address6"
```

Comment:
@rubailly The present story has a dependency on the load hierarchy PR where the mappings are defined for the address levels. For example, 'quartier' is mapped to 'address4' in https://github.com/jembi/cameroon-bahmni-metadata/pull/1 (see: https://github.com/jembi/cameroon-bahmni-metadata/blob/85d2dbb72b5b2215e33e2e38b13a7d0e1cb92516/configuration/addresshierarchy/addressConfiguration.xml#L42). I'd suggest we merge this story after the load hierarchy PR is merged (so that we can use the final mapping here).